Yehor/reactNetflix/src/pages/homePage/HomePage.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import Header from "../../components/header";
+import SearchPanel from "../../components/search-panel";
+import SortPanel from "../../components/sort-panel";
+import MoviesList from "../../components/moviesList";
+import Footer from "../../components/footer";
+import ErrorBlock from "../../components/errorBlock";
+import { RouteComponentProps } from "react-router-dom";
+import { stringify, parse } from "query-string";
+import IMovies from "../../interfaces/IMovies";
+
+type HomePageProps = IMovies & RouteComponentProps;
+
+class HomePage extends React.Component<HomePageProps> {
+  handleSearchChange = ({
+    search,
+    searchBy,
+  }: {
+    search: string;
+    searchBy: string;
+  }) => {
+    const query = stringify({ search, searchBy });
+    this.props.history.push({
+      pathname: "/",
+      search: query,
```

Comment:
you have to merge params with previous. If user click on sort and then use search form, you just replace sort url with new one.